### DISCUSSION TITLE
Difference between a Store and Transition Actor

### DISCUSSION THREAD

**Discussion by ninjz** [statelyai/xstate#4977] [Q&A]
I'm struggling to see the difference between a [Store](https://stately.ai/docs/xstate-store#quick-start) and [Transition](https://stately.ai/docs/xstate-store#quick-start) actor. They seem to pretty much be the same thing from what I can see besides the interfaces being a little different and the naming.

Is there a particular scenario where we should be using one over the other?

**Comment by davidkpiano** [comment]
They are pretty similar. Transition function actors are part of XState, whereas stores are something separate, but easily converted to an XState transition function actor. Also, a transition function is a bit more flexible, since it's just a plain function.
